src/scripts/main.js

```diff
@@ -1,3 +1,86 @@
 'use strict';
 
-// write your code here
+class DecimalFormat {
```

Comment:
oh.., let's not use classes for dom tasks

Reply:
Thank you. fixed